internal/auth0/resourceserver/resource_scope_test.go

```diff
@@ -21,61 +22,78 @@ resource "auth0_resource_server" "resource_server" {
 `
 
 const givenAScope = `
-resource "auth0_resource_server_scope" "read_posts" { 
-	scope = "read:posts"
+resource "auth0_resource_server_scope" "read_posts" {
 	resource_server_identifier = auth0_resource_server.resource_server.identifier
 
+	scope       = "read:posts"
 	description = "Can read posts"
 }
 `
 
 const givenAnUpdatedScope = `
-resource "auth0_resource_server_scope" "read_posts" { 
-	scope = "read:posts"
+resource "auth0_resource_server_scope" "read_posts" {
 	resource_server_identifier = auth0_resource_server.resource_server.identifier
 
+	scope       = "read:posts"
 	description = "Can read posts from API"
 }
 `
 
 const givenAnotherScope = `
-resource "auth0_resource_server_scope" "write_posts" { 
-	depends_on = [ auth0_resource_server_scope.read_posts ] 
+resource "auth0_resource_server_scope" "write_posts" {
+	depends_on = [ auth0_resource_server_scope.read_posts ]
 
-	scope = "write:posts"
 	resource_server_identifier = auth0_resource_server.resource_server.identifier
+
+	scope = "write:posts"
 }
 `
 
-const testAccNoScopesAssigned = givenAResourceServer
-const testAccOneScopeAssigned = givenAResourceServer + givenAScope + `data "auth0_resource_server" "resource_server" {
+const testAccNoScopesAssigned = `
```

Comment:
This was reusing the resource server with lifecycle ignore changes, which wasn't really accurately testing that the resource server has no scopes assigned as the value was ignored.